exporter/elasticsearchexporter/bulkindexer_test.go

```diff
@@ -596,3 +597,99 @@ func TestSyncBulkIndexer_flushBytes(t *testing.T) {
 		},
 	}, metricdatatest.IgnoreTimestamp())
 }
+
+func TestNewBulkIndexer(t *testing.T) {
+	for _, tc := range []struct {
```

Comment:
any tc for sending queue disabled, but `sending_queue::batch` is non nil? and if so, what do we expect to happen?

Reply:
> any tc for sending queue disabled, but sending_queue::batch is non nil?

In this case, the async bulk indexer will be used. Basically, `sending_queue::batch` will override the batcher w.r.t. the batching configuration, however, usage of async vs sync is purely dependent on the batching configuration being "enabled". Does that make sense?

BTW, that case is a missing case - I am adding it now.

Reply:
Done!